tests/functional/curate/cram/titlecase.t

```diff
@@ -3,16 +3,44 @@ Setup
   $ pushd "$TESTDIR" > /dev/null
   $ export AUGUR="${AUGUR:-../../../../bin/augur}"
 
+Test output with lowercase and already captialized strings.
 
-Create NDJSON file for testing titlecase with different forms
+  $ echo '{"record": 1, "authors": "john smith", "author2": "Jane Doe"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "authors" "author2"
+  {"record": 1, "authors": "John Smith", "author2": "Jane Doe"}
 
-  $ cat >$TMP/records.ndjson <<~~
-  > {"record": 1, "authors": "john smith", "author2": "Jane Doe"}
-  > ~~
+Test output with lowercase articles.
 
+  $ echo '{"record": 2, "title": "the night OF THE LIVING DEAD", "location": "BRAINE-LE-COMTE, FRANCE"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title" "location" --articles "a" "and" "of" "the" "le"
+  {"record": 2, "title": "The Night of the Living Dead", "location": "Braine-le-Comte, France"}
 
-Test output with Unicode normalization form "NFKC".
+Test output with abbreviations
 
-  $ cat $TMP/records.ndjson \
-  >   | ${AUGUR} curate titlecase --titlecase-fields "authors" "author2"
-  {"record": 1, "authors": "John Smith", "author2": "Jane Doe"}
+  $ echo '{"record": 3, "city": "Washington DC, USA"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "city" --abbreviations "USA" "DC"
+  {"record": 3, "city": "Washington DC, USA"}
+
+Test output with unicode characters
+
+  $ echo '{"record": 4, "location": "Auvergne-Rhône-Alpes" }' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "location"
+  {"record": 4, "location": "Auvergne-Rh\u00f4ne-Alpes"}
+
+Test output with numbers
+
+  $ echo '{"record": 5, "title": "2021 SARS-CoV"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title" --abbreviations "SARS"
+  {"record": 5, "title": "2021 SARS-Cov"}
+
+Test output with hyphens
+
+  $ echo '{"record": 6, "title": "the-night-of-the-living-dead"}' \
+  >   | ${AUGUR} curate titlecase --titlecase-fields "title"
+  {"record": 6, "title": "The-Night-Of-The-Living-Dead"}
+
+# Test output with apostrophes
+# 
+#   $ echo '{"record": 5, "title": "O\u0027Reilly\u0027s"}' \
+#   >   | ${AUGUR} curate titlecase --titlecase-fields "title"
+#   {"record": 5, "title": "O'Reilly's"}
```

Comment:
Added because I was looking for an edgecase (e.g. "Don't capitalize 's") but then the test failed returning "O'Reilly'S" and I hadn't committed to dig into it further. 

I'm leaning toward dropping this test case.

Reply:
Dropped